gf-guide/diacritics.md

```diff
@@ -17,70 +17,70 @@ Many diacritics are separated from the base letter and can be placed above, belo
-- Usually, the design of the diacritics needs to be adjusted in size or slope to work better with the uppercase letters. So, ideally, there should be at least two sets: lowercase and uppercase marks. E.g. `acutecomb` and `acutecomb.case`
+- You also need to create all the combining marks required for the language you are supporting, as well as the so-called "Legacy Marks". (See below.) Google Fonts requires at least the GF Latin Core as the minimum set for a font addressing the Latin script. Please read more about our <a herf="https://googlefonts.github.io/gf-guide/requirements.html#glyphsets">Glyphsets</a>.
+- Usually, the design of the diacritics needs to be adjusted in size or slope to work better with the uppercase letters. So ideally there should be at least two sets of diacritics: lowercase and uppercase marks. (e.g. `acutecomb` and `acutecomb.case`)
 
 ## Legacy – Spacing marks
 
-The spacing diacritical marks are required mainly for historical reasons and, therefore, for backward compatibility, which is why they are also known as Legacy marks.
+The spacing diacritical marks (for example, U+00B4 `ACUTE ACCENT`) are required mainly for historical reasons and for backward compatibility. This is why they are also known as "Legacy marks".
 
-They are only used as placeholder when typing for a combination of keys to add an accent to a base letter e.g. `´` + `a` to obtain a `á`. 
+These marks are only used as placeholder when typing for a combination of keys to add an accent to a base letter e.g. `´` + `a` to obtain a `á`. 
 
-**Conditions:**
+**Requirements::**
 
-- **They must have a positive width value `≠0`** Due to this function, they are expected to have an advanced width value with positive sidebearings, that is, they should not be zero width glyphs. 
+- **Legacy marks must have a positive advance width value `≠0`** Due to their function as displaying the accent as a standalone character, they are expected to have an advance width value with positive sidebearings, that is, they should not be zero width glyphs. 
 - They should share the design of the `combining marks` for consistency reasons. To ensure this in a practical way, it is suggested to create them by using the combining marks as components in the source file.
 - They use the simple name of the mark, e.g. `acute` or `grave`
-- They must have the right Unicode codepoint in your source file to work properly.
-- They belong to the mark-spacing category in the Glyphs Definiton of the `GDEF` table.
+- They must have the correct Unicode codepoint in your source file to work properly.
+- Although they represent marks, they need to behave as though they are base glyphs; that is, in the GDEF table their "Glyph Class" should be 1. ("Base glyph") In the Glyphs "Glyph info" pane, they should have `Category=Mark` and `Subcategory=Spacing`.
```

Comment:
pan -> panel ?

Reply:
Ah I confused with the so called "glyphs panel" in indesign. This is very specific to Glyphs App; is there a equivalent in Robofont and Fontlab?